webapp/src/components/ImageEditor/keyboardShortcuts.ts

```diff
@@ -235,4 +278,142 @@ export function replaceColor(fromColor: number, toColor: number) {
     const [ editState, type ] = currentEditState();
     const replaced = replaceColorEdit(editState, fromColor, toColor);
     dispatchAction(dispatchImageEdit(replaced.toImageState()));
+}
+
+function doReplace() {
+    const state = store.getState();
+
+    const fromColor = state.editor.backgroundColor;
+    const toColor = state.editor.selectedColor;
+
+    const [ editState ] = currentEditState();
+    const replaced = replaceColorEdit(editState, fromColor, toColor);
+    dispatchAction(dispatchImageEdit(replaced.toImageState()));
+}
+
+function doReplaceAllFrames() {
+    const state = store.getState();
+
+    const fromColor = state.editor.backgroundColor;
+    const toColor = state.editor.selectedColor;
+
+    editAllFrames(doReplace, editState => replaceColorEdit(editState, fromColor, toColor))
+}
+
+export function advanceFrame(forwards: boolean) {
+    const state = store.getState();
+
+    if (state.editor.isTilemap) return;
```

Comment:
Do we need to check if it's the image editor as well?

Reply:
that's what the frames length check does! the image editor uses the same state as the animation editor, it just has only one frame